agrona/src/main/java/org/agrona/collections/IntObjConsumer.java

```diff
@@ -19,7 +19,8 @@
  * This is an (int, Object) primitive specialisation of a BiConsumer.
  */
 @FunctionalInterface
-public interface IntObjConsumer<T>
+public interface
```

Comment:
We should retain the original formatting here.

Reply:
unfortunately SpecialisationGenerator will only generate LongObjConsumer if it is formatted to separate line, just like [IntIntConsumer](https://github.com/real-logic/agrona/blob/master/agrona/src/main/java/org/agrona/collections/IntIntConsumer.java)